gradle/asciidoc.gradle

```diff
@@ -125,15 +125,15 @@ configure(rootProject) {
 		mustRunAfter "generateObservationDocs"
 		from(project.rootProject.buildDir.toString() + "/generatedMetricsDocs/meterListener/") {
 			include "_*.adoc"
-			rename '_metrics.adoc', 'meterListener.adoc'
+			rename '_(.*).adoc', 'meterListener_$1.adoc'
```

Comment:
without this pattern-based renaming, generated documents now include another file which leads to trying to copy 3 similarly named files into a single directory => copy task fails